### ISSUE TITLE
Cannot style axis (textFill, etc.)

### ISSUE THREAD

**Issue by DonChatelain** [shauns/react-d3-radar#11]
Looks to me that the {style} prop is not being passed into RadarAxis from RadarWrapper

**Comment by BWalti** [comment]
yes, double that! https://github.com/shauns/react-d3-radar/pull/21
but as this issue has been raised in 2018 and the fix would be trivial, I assume this will never be merged.. :(
